### DISCUSSION TITLE
Stimulus Variant for Ruby on Rails users

### DISCUSSION THREAD

**Discussion by gobijan** [tailwindlabs/headlessui#1200] [Ideas]
Hi,

as Tailwind was promoted to a first class option for new Rails applications it would be nice to have headlessui support it's default JS Framework called [stimulus](https://stimulus.hotwired.dev/)
As a tailwindui customer I would be more than happy to be able to use the dynamic components out of the box by copy pasting.
Is this on your roadmap?

Cheers Bijan

**Comment by pySilver** [comment]
+1

**Comment by philliplongman** [comment]
This is something I'd love to see as well.
